File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_1/exercice12.py
M=float(input("saisir la note de Math: "))
PH=float(input("saisir la note de Physique: "))
FR=float(input("saisir la note de Français: "))
AN=float(input("saisir la note de Anglais: "))
GEO=float(input("saisir la note de Géographie: "))
C_M,C_PH,C_FR,C_AN,C_GEO=5,5,4,2,3
somme=M*C_M + PH*C_PH + FR*C_FR +AN*C_AN + GEO*C_GEO
moyenne=somme/19
print("le moyenne est: ",moyenne)

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_1/exercice12.c
#include <stdio.h>
float  M,PH,FR,AN,GEO,moyenne,somme;
int C_M=5,C_PH=5,C_FR=4,C_AN=2,C_GEO=3;
int main(){
    printf("sasir la note de Math: ");
    scanf("%f",&M);
    printf("sasir la note de Physique: ");
    scanf("%f",&PH);
    printf("sasir la note de Français: ");
    scanf("%f",&FR);
    printf("sasir la note de Anglais: ");
    scanf("%f",&AN);
    printf("sasir la note de Géographie: ");
    scanf("%f",&GEO);
    somme=M*C_M + PH*C_PH + FR*C_FR +AN*C_AN + GEO*C_GEO;
    moyenne=somme/19;
    printf("le moyenne est: %f",moyenne);
    return 0;
}